### DISCUSSION TITLE
seperation

### DISCUSSION THREAD

**Discussion by ahmedcc12** [CarlGao4/Demucs-Gui#39] [Q&A] [answered]
i was wondering if we could specify what we want to separate , for example only separate the drums from the rest of the track

**Comment by CarlGao4** [comment, marked as answer]
This function is still under development, will add it in 1.1

**Comment by 1GR3** [comment]
+1 I'd like to convert some music to karaoke instrumentals and mixing down separate channels seems completely unnecessary.

**Comment by dts350z** [comment]
+1 for support of --two-stems drums

to separate, say, just drums from everything else. Super helpful if also runnable from a windows batch or git bash script (in the background).
